.github/workflows/lerna.yaml

```diff
@@ -20,3 +20,7 @@ jobs:
         run: npx lerna run test
       - name: Lint
         run: npx lerna run lint
+      - name: Generate ABIs
```

Comment:
added these steps to the existing lerna pipeline instead of making a separate workflow. i think it should be faster this way because it doesn't have to install all of the npm packages in a separate thing, but def open to doing it a different way.